test.md

```diff
@@ -0,0 +1 @@
+"This should be ignored by git"
```

Comment:
@anagudelogu  Thank you for for your most helpful feedback. I had forgotten to the test.md file to the .gitignore file earlier on but it's the last thing that I did, could you please check to see if indeed it is appearing , or should I write the actual test instead of the test.md?

Reply:
Hi @Leeoasis, it is indeed appearing:

![imageedit_2_2048671953](https://user-images.githubusercontent.com/77069011/211619925-b87f006f-38b3-4c78-9813-73e823aff145.gif)

If you need a little guidance on how to make the change, you could check [this article](https://www.atlassian.com/git/tutorials/saving-changes/gitignore#:~:text=If%20you%20want%20to%20ignore,directory%20as%20an%20ignored%20file.).